test/Controllers/PricingModelsTest.php

```diff
@@ -0,0 +1,181 @@
+<?php
+/*
+ * PricingModels SDK Controller Test Case
+ */
+
+namespace GonebusyTest\Controllers;
+
+use PHPUnit\Framework\TestCase;
+
+use GonebusyLib\Configuration;
+use GonebusyLib\GonebusyClient;
+use GonebusyLib\Models\CreatePricingModelBody;
+use GonebusyLib\Models\UpdatePricingModelByIdBody;
+
+class PricingModelsTest extends TestCase
+{
+    /**
+     * To contain the GonebusyLib\GonebusyClient
+     */
+    protected $client;
+
+    /**
+     * To contain the GonebusyLib\Controllers\PricingModelsController
+     */
+    protected $pricingModels;
+
+
+    /**
+     * Create the GonebusyClient and PricingModelsController for each test.
+     */
+    public function setUp() {
+        $this->client = new GonebusyClient();
+        $this->pricingModels = $this->client->getPricingModels();
+    }
+
+
+    /**
+     * Generate unique pricing model data.
+     * @param  string $type Should be 'create' or 'update'.
+     * @return  CreatePricingModelBody or UpdatePricingModelByIdBody object with unique data to send to API
+     */
+    private function pricingModelBody($type) {
+        switch($type) {
+            case 'create':
+                return new CreatePricingModelBody(
+                    "name",
+                    "Activity", // type
+                    "USD", // currency XXX This isn't having effect.
```

Comment:
Done. And yes I mostly add codetags so we notice them in the PRs! But I guess it's not the best system, I can just note it here directly without he code comments.